legal-api/tests/conftest.py

```diff
@@ -267,25 +268,16 @@ def session(app, db):  # pylint: disable=redefined-outer-name, invalid-name
         txn = conn.begin()
 
         try:
-            options = dict(bind=conn, binds={})
-            # sess = db.create_scoped_session(options=options)
-            sess = db._make_scoped_session(options=options)
+            sess = scoped_session(
```

Comment:
Looks good, much better than what I wrote lol

Reply:
https://github.com/bcgov/sbc-pay/blob/06673215d07ac896fdc84e134f424de4d89c87b5/pay-api/tests/conftest.py#L84

I'll give it a go in pay